hw/ip/otbn/dv/otbnsim/sim/insn.py

```diff
@@ -297,6 +305,14 @@ def __init__(self, raw: int, op_vals: Dict[str, int]):
         self.csr = op_vals['csr']
         self.grs1 = op_vals['grs1']
 
+    def pre_execute(self, state: OTBNState) -> bool:
+        if (self.csr == 0xfc0):
```

Comment:
These parentheses aren't needed. Also, I think this should probably be dependent on `self.grd != 0`, right?